Parsers/ASimRegistry/RegistryEventEmpty.yaml

```diff
@@ -0,0 +1,74 @@
+Parser:
+  Title: ASIM Empty Registry Event Table
+  Version: '0.1.0'
+  LastUpdated: June 29, 2021
+Product:
+  Name: Source Agnostic
+Normalization:
+  Schema: RegistryEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Registry Schema
+  Link: https://aka.ms/AzSentinelRegistryEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AzSentinelNormalization
+Description: |
+  The purpose of this KQL function is to generate and guarentee the schema columns for the ASIM Registry Event schema
```

Comment:
misspelled - guarantee - but maybe we should say this in a different way such as "make available the default schema" or something like that.

Reply:
Changed per your suggestion